src/pages/docs/connect/states.mdx

```diff
@@ -409,3 +409,19 @@ While an SDK will not automatically attempt to reconnect in the `FAILED` state,
 Other classes of error are nonfatal. For example, a client may have network connectivity issues. An SDK will attempt to automatically reconnect and recover from these sort of issues, as detailed in the `DISCONNECTED` and `SUSPENDED` explanations in the [Available connection states](#connection-states) section.
 
 If message continuity is lost in the process, e.g. because you have been disconnected from Ably for more than two minutes, the SDK will notify you through the [`resumed`](#resume) flag mechanism.
+
+<If lang="swift,objc">
+
+### iOS connection errors <a id="iOS-errors"/>
+
+When using Ably in mobile apps, the connection will attempt to stay active even when your app moves to the background. However, iOS may terminate background network connections to preserve battery life, especially when the app is suspended or force-quit.
+
+The Ably SDK will continue attempting to reconnect automatically, but iOS progressively limits background network activity.
+
+If you want to explicitly <If lang="swift">[manage connections](/docs/connect/states?lang=swift)</If> [manage connections](/docs/connect/states?lang=swift) <If lang="objc">[manage connections](/docs/connect/states?lang=objc)</If> based on your app's lifecycle:
+
+- Call <If lang="swift">[`close()`](/docs/api/realtime-sdk/connection?lang=swift#close)<If lang="objc">[`close()`](/docs/api/realtime-sdk/connection?lang=objc#close)</If> </If> when the app enters background if you don't need background connectivity.
+- Call <If lang="swift">[`connect()`](/docs/api/realtime-sdk?lang=swift#connect)</If> <If lang="objc">[`connect()`](/docs/api/realtime-sdk?lang=objc#connect)</If> when the app returns to foreground to re-establish the connection.
```

Comment:
You don't need the if statements here, because the language will be in your URL already so it'll correctly display in the correct language. I think you're also missing a "the".